droid-command-line/src/main/java/uk/gov/nationalarchives/droid/command/action/CommandLineParam.java

```diff
@@ -1,505 +1,517 @@
+    public String toString() {
+        return shortName;
+    }
+
+    /**
+     *
+     * @return a new option for the parameter.
+     */
+    Option newOption() {
+        Option option = new Option(shortName, longName, maxArgs != 0, I18N.getResource(resourceKey));
+        option.setArgs(maxArgs == -1 ? Option.UNLIMITED_VALUES : maxArgs);
+        if (maxArgs != 0) {
+            option.setOptionalArg(!argsRequired);
+        }
+        option.setArgName(argName);
+
+        return option;
+    }
+
+
+    private static String filters() {
+        return "filter ...";
+    }
+
+    /**
+     * All options.
+     * @return all options
+     */
+    public static Options options() {
+        Options options = new Options();
+
+        OptionGroup topGroup = new OptionGroup();
+
+        for (CommandLineParam param : TOP_LEVEL_COMMANDS.values()) {
+            topGroup.addOption(param.newOption());
+        }
+
+        options.addOption(PROFILES.newOption());
+        options.addOption(REPORT_NAME.newOption());
+        options.addOption(REPORT_OUTPUT_TYPE.newOption());
+        options.addOption(SIGNATURE_FILE.newOption());
+        options.addOption(CONTAINER_SIGNATURE_FILE.newOption());
+        options.addOption(EXTENSION_LIST.newOption());
+        options.addOption(ARCHIVES.newOption());
+        options.addOption(ARCHIVE_TYPES.newOption());
+        options.addOption(WEB_ARCHIVES.newOption());
+        options.addOption(WEB_ARCHIVE_TYPES.newOption());
+        options.addOption(RECURSIVE.newOption());
+        options.addOption(QUIET.newOption());
+        options.addOption(BOM.newOption());
+
+        OptionGroup filterOptions = new OptionGroup();
+        filterOptions.addOption(ALL_FILTER.newOption());
+        filterOptions.addOption(ANY_FILTER.newOption());
+
+        options.addOptionGroup(filterOptions);
+        options.addOptionGroup(topGroup);
+
+        return options;
+
+    }
+
+    /**
+     * Single Options.
+     *
+     * @return the Single Options
+     */
+    public static Options singleOptions() {
+        Options options = new Options();
+
+        options.addOption(CHECK_SIGNATURE_UPDATE.newOption());
+        options.addOption(DOWNLOAD_SIGNATURE_UPDATE.newOption());
+        options.addOption(HELP.newOption());
+        options.addOption(LIST_REPORTS.newOption());
+        options.addOption(CONFIGURE_DEFAULT_SIGNATURE_VERSION.newOption());
+        options.addOption(VERSION.newOption());
+        options.addOption(DEFAULT_SIGNATURE_VERSION.newOption());
+        options.addOption(LIST_SIGNATURE_VERSIONS.newOption());
+
+        return options;
+    }
+
+    /**
+     * No Profile Run sub-options.
+     *
+     * @return sub-options for no-profile run
+     */
+    public static Options noProfileRunSubOptions() {
+        Options options = new Options();
+
+        Arrays.asList(SIGNATURE_FILE, CONTAINER_SIGNATURE_FILE, EXTENSION_LIST, ARCHIVES, WEB_ARCHIVES, WEB_ARCHIVE_TYPES, RECURSIVE, QUIET).forEach(
```

Comment:
missing the newly added `ARCHIVE_TYPES`

Reply:
added in https://github.com/digital-preservation/droid/pull/356/commits/e3f376bb3eaa2d44d4ff9a397952b7a5f81d47fe